packages/victory-core/src/victory-primitives/background.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import PropTypes from "prop-types";
+import CommonProps from "../victory-util/common-props";
+import Rect from "./rect";
+import Circle from "./circle";
+
+const getRangeBounds = (props) => ({ x: props.range.x[0], y: props.range.y[1] });
```

Comment:
I'm on the fence about whether these positioning attributes should be calculated here or by `VictoryChart`. I'm trying to think about what would be easiest for users who want to override props on the background component. I think that pushes me in the direction of wanting the calculations to be taken care of by `VictoryChart` instead. I think this is also more consistent with other primitive components.

Reply:
Agreed. Would it be consistent w/ other primitives if there were optional props for backgroundX, backgroundY, backgroundHeight, and backgroundWidth on VictoryChart, and the defaults were the values calculated here?

Reply:
Agreed. Would it be consistent w/ other primitives if there were optional props for backgroundX, backgroundY, backgroundHeight, and backgroundWidth on VictoryChart, and the defaults were the values calculated here?